index.html

```diff
@@ -75,17 +75,15 @@ <h2 class="section-title">About Me</h2>
            
             <p>
               I am <strong> Roselida Akoth Ochieng'</strong>
```

Comment:
Paragraph tags are block level elements that cannot contain other block level elements. You'll want to close this `<p>` tag here.